include/vcpkg/base/messages.h

```diff
@@ -510,6 +510,33 @@ namespace vcpkg
                     "An example of env_var is \"HTTP(S)_PROXY\""
                     "'--' at the beginning must be preserved",
                     "-- Automatically setting {env_var} environment variables to \"{url}\".");
+    DECLARE_MESSAGE(BinParagraphAbi,
+                    (msg::value),
+                    "'{value}' is the value of abi in the binary paragraph.",
+                    "\nabi: '{value}'");
+    DECLARE_MESSAGE(BinParagraphDefaultFeature,
+                    (msg::value),
+                    "'{value}' is the binary paragraph's default feature.",
+                    "\ndefault_feature: ['{value}']");
+    DECLARE_MESSAGE(BinParagraphDependencies,
+                    (msg::value),
+                    "'{value}' is the binary paragraph's dependencies.",
+                    "\ndependencies: ['{value}']");
+    DECLARE_MESSAGE(BinParagraphDescription,
+                    (msg::value),
+                    "'{value}' is the binary paragraph's description.",
+                    "\ndescription: ['{value}']");
+    DECLARE_MESSAGE(BinParagraphFeature,
+                    (msg::value),
+                    "'{value}' is the value of feature in the binary paragraph.",
+                    "\nfeature: '{value}'");
+    DECLARE_MESSAGE(BinParagraphMaintainers,
+                    (msg::value),
+                    "'{value}' is the binary paragraph's maintainers.",
+                    "\nmaintainers: ['{value}']");
+    DECLARE_MESSAGE(BinParagraphPortVersion, (msg::version), "", "\nport_version: {version}");
+    DECLARE_MESSAGE(BinParagraphSpecAndVersion, (msg::spec, msg::version), "", "\nspec: '{spec}'\nversion: {version}");
+    DECLARE_MESSAGE(BinParagraphType, (msg::value), "'{value}' is the binary paragraph type", "\ntype: '{value}'");
```

Comment:
These are intended to be a dump of an on disk file for us to review so I don't think we should localize them.